week7/LeeHyeonSeung/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+import MenuGrid from "../../components/MenuGrid/MenuGrid";
+import OrderBar from "../../components/OrderBar/OrderBar";
+
+const Home = () => {
+  return (
+    <Wrapper>
+      <Title>오늘은 무엇을 먹을까요?</Title>
+      <Address>한남중앙로 40길 (한남 빌리지)으로 배달 &gt;</Address>
+
+      <MenuGrid />
+      <Spacing />
+      <OrderBar />
+    </Wrapper>
+  );
+};
+
+export default Home;
+
+const Wrapper = styled.div`
+  padding-bottom: 120px;
+  width: 100%;
+  min-height: 100vh;
+`;
+
+const Title = styled.h2`
+  font-family: Pretendard;
+  font-weight: 700;
+  font-size: 26px;
+  line-height: 100%;
+  letter-spacing: 0px;
+  color: #191F28;
+  margin: 0;
+  margin-top: 70px;
+  margin-left: 24px;
```

Comment:
margin은 숫자 4개를 써주면 순서대로 상 우 하 좌에 적용되기 때문에 한 줄에 쓸 수도 있습니다.
```suggestion
margin: 70px 0 0 24px;
```